src/components/Schedule/ProgressBar.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+interface IProps {
+  actual: number;
+  max: number;
+}
+
+export const ProgressBar: FC<IProps> = ({ actual, max }) => {
+  if (actual === 0) {
+    return null;
+  }
+  return (
+    <div
+      style={{ width: (actual / max) * 100 + "%" }}
+      className="border-b border-2 border-black absolute"
```

Comment:
according to the desing, the black line should have rounded corners, the same with grey background